emily/handler/src/api/handlers/deposit.rs

```diff
@@ -439,19 +440,32 @@ pub async fn update_deposits(
         // in order to enforce added stability to the API during a reorg.
         let api_state = accessors::get_api_state(&context).await?;
         api_state.error_if_reorganizing()?;
+
+        let is_trusted_key = context.settings.trusted_reorg_api_key == api_key;
+        // Signers are only allowed to update deposits to the accepted state.
+        if !is_trusted_key {
```

Comment:
If we get here in the code, then we know that it is one of the signers or one of the trusted callers. If `is_trusted_key` is false then we know that it is one of the signers. Is that correct?